cpp/tests/utilities/column_utilities.cu

```diff
@@ -1038,13 +1038,40 @@ struct column_view_printer {
     if (col.is_empty()) return;
     auto h_data = cudf::test::to_host<std::string>(col);
 
+    // explicitly replace '\r' and '\n' characters with "\r" and "\n" strings respectively.
+    auto cleaned = [&](std::string const& in) {
```

Comment:
This function feels kind of heavy. Here are alternative approaches. Look at the bottom one for the solution I recommend.
- Thrustless (starting point for Godbolt manipulations -- not necessarily better) https://godbolt.org/z/aq44GqYcK
- First attempt to use `std::string::replace` https://godbolt.org/z/G7Yqhv6zd
- Minimal solution using `std::string::replace`: 14 LOC https://godbolt.org/z/sPc8jzsMd

Reply:
Hah.  I guess I did kind of overengineer this :)

Reply:
Unfortunately, the minimal solution I proposed is only half as fast as the "Thrustless" adaptation I posted above. 
https://quick-bench.com/q/c2gdYf1RVFu86xZF8xpSTeKFRlQ

Feel free to proceed however you like here.

Reply:
Is there any way to adopt some external tool like `fmt` so we just pass in our data and get out the desired output?

Reply:
So we've got two options here.  Speed (my original code) or compactness (@davidwendt s solution below).  I'm fine with either one.  Just tell me which you want.

Reply:
Because this is host-side testing code, I am inclined to use @davidwendt’s proposal. Cleanliness is important here because we might want to do the same thing for `\t` or other specific characters in the future.